core/src/main/java/nl/nn/adapterframework/jms/JmsSender.java

```diff
@@ -300,7 +306,7 @@ private void setProperties(javax.jms.Message msg, ParameterValueList pvl) throws
 
 			if ((!isSoap() || !name.equals(getSoapHeaderParam()) && (StringUtils.isEmpty(getDestinationParam()) || !name.equals(getDestinationParam())))) {
 
-				if (log.isDebugEnabled()) { log.debug(getLogPrefix()+"setting ["+type+"] property from param ["+name+"] to value ["+property.getValue()+"]"); }
+				if (log.isDebugEnabled()) { log.debug("{} setting [{}] property from param [{}] to value [{}]", getLogPrefix(), type, name, property.getValue()); }
```

Comment:
```suggestion
				log.debug("{} setting [{}] property from param [{}] to value [{}]", this::getLogPrefix, ()->type, ()->name, property::getValue);
```

Schoont aardig op zo. De logPrefix bevat niet veel extra info meer, mocht je een idee hebben om de naam op een mooie manier in de mdc te krijgen oid hoor ik het graag!

Reply:
Suggestie overgenomen. MDC-ideetje parkeer ik effe.